_posts/2023-01-23-there-is-no-cloud.md

```diff
@@ -0,0 +1,294 @@
+# There Is No Cloud, It's Just Someone Else's Computer
+
+### By Sergei Vorobyov, Solita/Stockholm
+
+This well-known adage unfortunately isn't a joke, and you should
+clearly understand who actually owns (has access to all) your cloud
+data. In most cases, no matter what you do, Big Brother easily
```

Comment:
who is big brother here? the government?

Reply:
Your competitor, enemy, cloud provider, governments

Reply:
Author line deleted